morphia/src/main/java/org/mongodb/morphia/annotations/Indexed.java

```diff
@@ -76,10 +76,7 @@
     boolean unique() default false;
 
     /**
-     * Indicates the direction of the index (ascending, descending); default is ascending
-     *
-     * @deprecated use the {@link IndexOptions} found in {@link #options()}
+     * Indicates the type of the index (ascending, descending); default is ascending
      */
-    @Deprecated
```

Comment:
It probably was released as deprecated.  This particular annotation isn't used as much so it doesn't seem as if anyone ran in to it.